README.md

```diff
@@ -54,6 +54,7 @@ class MyExampleViewSet(XLSXFileMixin, ReadOnlyModelViewSet):
     serializer_class = MyExampleSerializer
     renderer_classes = (XLSXRenderer,)
     filename = 'my_export.xlsx'
+    pagination_class = None # if you have custom pagination, you need to set this to None
```

Comment:
Inline comments should be at least two spaces after the code: https://peps.python.org/pep-0008/#inline-comments

I'll take care of this. Thanks for your sleuthing!